qiskit_experiments/curve_analysis/standard_analysis/error_amplification_analysis.py

```diff
@@ -72,7 +72,7 @@ class ErrorAmplificationAnalysis(curve.CurveAnalysis):
                 where a is given by :code:`max(abs(angle_per_gate), np.pi / 2)`.
                 Extra guesses are added based on curve data when either :math:`\rm amp` or
                 :math:`\rm base` is :math:`\pi/2`. See fit model for details.
-            bounds: [-pi, pi].
+            bounds: [-0.8 pi, 0.8 pi].
```

Comment:
```suggestion
            bounds: [-0.8 pi, 0.8 pi]. The bounds do not include plus and minus pi since these values
                often correspond to symmetry points of the fit function. Furthermore, this type of analysis
                is intended for values of :math:`d\theta` close to zero.
```